tests/test_api.py

```diff
@@ -84,11 +84,27 @@ def test_upload_attachment(self) -> None:
             api.upload_attachment(
                 TEST_PAGE_ID,
                 self.sample_dir / "figure" / "interoperability.png",
-                "figure_interoperability.png",
-                "A sample figure",
+                "figure/interoperability.png",
```

Comment:
As far as I recall, this was fixed on the branch `master`, and names are now sanitized to replace characters problematic for Confluence with `_`.